src/styles/utils/common.scss

```diff
@@ -0,0 +1,5 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
it's better to use only box-sizing: border-box; inside * selector

Reply:
done